src/components/removeButton.jsx

```diff
@@ -0,0 +1,16 @@
+import PropTypes from 'prop-types';
+
+const RemoveButton = (props) => {
+  const { onClick } = props;
+  return (
+    <button type="button" onClick={onClick}>
+      Remove
+    </button>
```

Comment:
- [ ] You have added the event handler but the removeBook action(with the id attribute) is not being dispatched to the redux store as requested.


 <Button onClick={() => dispatch(removeBook(id))}>Remove</Button>